README.md

```diff
@@ -2,26 +2,24 @@
 
 Welcome to the Nava Platform! This is the home repository for Nava's Platform tools that empowers teams to accelerate their delivery through self-service tools and training.
 
-To learn more about the platform, check out our [Platform space on Nava Sage](https://navasage.atlassian.net/wiki/spaces/PLAT/overview).
+## Platform Templates
 
-## Application Templates
-
-Application templates allow project teams to quickly spin up new production ready applications. Engineers can copy the templates to their own project's GitHub repository in order to get started more quickly. The templates provide a lot of default technical decisions that represent Nava's best practices.
+Platform templates allow project teams to quickly spin up new production ready applications. Engineers can copy the templates to their own project's GitHub repository in order to get started more quickly. The templates provide many default technical decisions that represent best practices for implementing government systems. These best practices were evolved by Nava over many years of experience working on GovTech projects, and will continue to evolve.
```

Comment:
Not required, but you could mentioned one of the success metric results, i.e. how X template was used to spin up Y project in Z time, given it could further pique the interest of the USDS audience.